src/plugins/azgenerator/TemplateAzureCliCustom.ts

```diff
@@ -136,7 +128,7 @@ function GetCommandBody(model: CodeModelAz, required: boolean, needUpdate: boole
         // create body transformation for methods that support it
         let methodName: string = model.Command_MethodName;
 
-        if (methodName != "show" && methodName != "list" && methodName != "delete") {
+        //if (methodName != "show" && methodName != "list" && methodName != "delete") {
             // body transformation
```

Comment:
please remove the code if it's not needed

Reply:
removed